docs/ref/tasks.txt

```diff
@@ -0,0 +1,318 @@
+=====
+Tasks
+=====
+
+.. module:: django.tasks
+   :synopsis: Django's built-in background task system.
+
+Task definition
+===============
+
+.. function:: task
+
+A decorator defining a :class:`Task`::
+
+    from django.tasks import task
+
+
+    @task
+    def calculate_meaning_of_life() -> int:
+        return 42
```

Comment:
The `calculate_meaning_of_life` is the main documented example. I think it would make sense to have a more realistic example to help demonstrate when you would use a task. So I think we should have either the example to always be sending an email (or something similar), or the main examples to be sending emails (and then others can be simpler)